### PR TITLE
Clarify that code has to be added in the "Create a template for the post details" section

### PR DESCRIPTION
**Pull request by deeptikorwar**
Clarified that code has to be added in the "Create a template for the post details" section.

Changes in this pull request:

- modified line 137
-
-

### CODE REVIEW

File: en/css/README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Go ahead and name some parts of the HTML code. Add a class called `page-header`
 </div>
 ```
 
-And now add a class `post` to your `div` containing a blog post.
+And now add a class `post` to your `div` that displays each blog post details.
```
Comment (das-g) [inline en/css/README.md]: This seems grammatically wrong. It also might be semantically confusing, as this is the view for an overview over all published blog posts, not the view to show a post's _details_. (Though, obviously, _some_ details are shown in the overview, too.)

**Comment by deeptikorwar** [comment]
Hi das-g, 

Is there anything else I need to add to this pull request for the re-review?

Thanks,
Deepti

**Comment by das-g** [comment]
> Is there anything else I need to add to this pull request for the re-review?

Sorry for letting you wait. I'll have a look at it now.

File: en/extend_your_application/README.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ Oh no! Another error! But we already know how to deal with it, right? We need to
 
 We will create a file in `blog/templates/blog` called `post_detail.html`, and open it in the code editor.
 
-It will look like this:
+Let's type the following code:
```
Comment (das-g) [inline en/extend_your_application/README.md]: :+1:

File: en/html/README.md [hunk 3]
```diff
@@ -208,6 +208,9 @@ $ git pull
 
 You'll need to substitute `<your-pythonanywhere-domain>` with your actual PythonAnywhere subdomain name, without the angle-brackets. Your subdomain name is normally your PythonAnywhere user name, but in some cases it might be a bit different (such as if your user name contains capital letters). So if this command doesn't work, use the `ls` (list files) command to find your actual subdomain/folder name, and then `cd` to there.
 
+Note that if you are still in the same Bash console used previously in this tutorial, you don’t have to use the `cd` command as you would already be in that directory.
```
Comment (das-g) [inline en/html/README.md]: This seems unrelated to the other change or to the stated purpose of your pull request, so please file a separate pull request about this (and remove this change from this pull request), so it can be considered independently from the other change.

Comment (deeptikorwar) [reply en/html/README.md]: Sure. I'll delete this file from the PR now.

File: en/extend_your_application/README.md (hunk 2 above)
Comment (deeptikorwar) [reply en/extend_your_application/README.md]: Thank you. I have taken out the other file and edited the description of this PR. Please re-review.
